conformance/lang/expressions/unary-expression/unary_minus_expression.balt

```diff
@@ -513,27 +513,44 @@ function init() {
    float a = 1.0;
    float b = 101.5;
    float c = 0;
-   float d = 922439289.0;
 
    io:println(-a); // @output -1.0
    io:println(-b); // @output -101.5
    io:println(-c); // @output -0.0
-   io:println(-d); // @output -9.22439289E8
 }
 
+Test-Case: output
+Description: Test unary negative expression for large positive floats with no overflow scenarios.
+Fail-Issue: ballerina-platform/ballerina-spec#1046
+Labels: DecimalFloatingPointNumber, float, unary-minus
+
+function init() {
+   float a = 922439289.0;
+
+   io:println(-a); // @output -9.22439289E8
```

Comment:
Shall we change the output to the correct output?

Reply:
fixed